exporters/otlp/CMakeLists.txt

```diff
@@ -33,7 +33,14 @@ if(BUILD_TESTING)
     TARGET recordable_test
     TEST_PREFIX exporter.otlp.
     TEST_LIST recordable_test)
-  find_library(GMOCK_LIB gmock PATH_SUFFIXES lib)
+  if(MSVC)
+    add_definitions(-DGTEST_LINKED_AS_SHARED_LIBRARY=1)
```

Comment:
Perhaps `target_compile_definitions` to make the definition target specific?

Reply:
If we add more tests, unit tests, or benchmarks, then this definition would apply to all other projects in this directory and subdirectories, so then we won't have to do  `target_compile_definition` every time. In that sense `add_definitions` is a bit more universal.

Reply:
I am thinking `MAYBE` we should even pull it at the top-most level as a build option - that detects whether Google Test and Google Benchmark and Google Mock are installed as DLLs, but I don't want to touch it right now. 😄